src/main/java/com/github/wovnio/wovnjava/WovnServletFilter.java

```diff
@@ -89,22 +89,22 @@ private void tryTranslate(Headers headers, RequestOptions requestOptions, HttpSe
             wovnRequest.getRequestDispatcher(headers.getCurrentContextUrlInDefaultLanguage().getPath()).forward(wovnRequest, wovnResponse);
         }
 
-        String originalBody = wovnResponse.toString();
-        if (originalBody != null) {
```

Comment:
IS that OK to remove this condition?

Reply:
If you check the implementation of this code, there are essentially no cases where the constructor can return a null, as in Java, it is not allowed to return a null.
Therefore, the original code is not working at all.
https://github.com/WOVNio/wovnjava/blob/master/src/main/java/com/github/wovnio/wovnjava/Utf8.java#L34